### DISCUSSION TITLE
Duplicating an entry should add a "- Copy {num}" suffix

### DISCUSSION THREAD

**Discussion by Antimated** [craftcms/cms#13565] [Ideas] [answered]
Not sure if this is a feature already because I don't seem to find any info about this in the docs but there should be a way (perhaps a general config setting) to add a "- Copy {num}" suffix to duplicated entries, categories, ...

I've had clients that duplicated content and are then editing the original instead of the duplicated entry.  Later they removed the original (in my case a category that was linked to a lot of other entries) which broke a lot of links on the site. Luckily I could fix this by reverting the trashed item. Still it would be nice to know what entry is the actual duplicated entry :)!

**Comment by cherrykoda** [comment, marked as answer]
This was asked awhile ago, but in Craft 5 they've changed copies to create drafts instead of live entries, in case this helps!

**Reply by brandonkelly** [reply]
Agree, that change makes this request moot.
